docs/ref/models/expressions.txt

```diff
@@ -181,7 +181,8 @@ instance and will be applied on each :meth:`~Model.save()`. For example::
     reporter.save()
 
 ``stories_filed`` will be updated twice in this case. If it's initially ``1``,
-the final value will be ``3``.
+the final value will be ``3``. This persistence can be avoided simply by
```

Comment:
Please remove "simply" — it's not appropriate.

Reply:
Thanks, fixed.